apps/src/blockly/addons/cdoBlockSvg.js

```diff
@@ -26,9 +26,7 @@ export default class BlockSvg extends GoogleBlockly.BlockSvg {
   isUnused() {
     const isTopBlock = this.previousConnection === null;
     const hasParentBlock = !!this.parentBlock_;
-    const isDragging =
-      this.workspace?.currentGesture_?.blockDragger_?.draggingBlock_ === this;
-    return isDragging || !(isTopBlock || hasParentBlock);
+    return !(isTopBlock || hasParentBlock);
```

Comment:
No, that expression is still correct. That code gets called when the block drag ends (the block is about to be dropped). However, `isDragging` was still `true` (because the drag hadn't technically finished ending yet), so the block was always being set to not enabled